lib/ohai/plugins/linux/network.rb

```diff
@@ -81,8 +81,16 @@ def check_routing_table(family, iface, default_route_table)
       logger.trace("Plugin Network: Parsing #{line}")
       if /\\/.match?(line)
         parts = line.split("\\")
-        route_dest = parts.shift.strip
+        dest_parts = parts.shift.strip.split("\s", 2)
```

Comment:
don't specify `\s`, just use the default splitting which is smart whitespace split.